src/Domain/FundRepository.php

```diff
@@ -103,6 +103,13 @@ public function pullForCampaign(Campaign $campaign): void
 
                     $campaignFunding->setAmount($amountForCampaign);
                 }
+
+                if (bccomp($increaseInAmount, '0.00', 2) === -1) {
+                    $this->logger->error(
+                        "Funding ID {$campaignFunding->getId()} balance could not be negative-increased by " .
+                        "£{$increaseInAmount}"
```

Comment:
Probably also worth logging the SF Fund ID here since I guess this will generally happen after someone has edited that fund? I haven't tested but I think it would be something like this:

```suggestion
                        "Funding ID {$campaignFunding->getId()} balance could not be negative-increased by " .
                        "£{$increaseInAmount}" + " SF Fund ID: " + $fundData['id']
```

Reply:
Yep, good idea. Did that with slightly different syntax in https://github.com/thebiggive/matchbot/pull/1198/commits/23454e7e3b75e007a5e4d46fd30562ea763a5908